docs/DeveloperGuide.md

```diff
@@ -81,6 +81,19 @@ the user input and execute the commands accordingly. The diagram below depicts t
 
 **Command**
 
+**PinCommand**
+
+1. Created by the parser function
```

Comment:
Don't think the PinCommand is created by the parser function right? Maybe can phrase it better?

Reply:
Instantiated by the parser function?

Reply:
Yeap I guess! But this one can leave out for now. Jun Xiang will be modifying the Parser command to make our code reflect more OOP